cli/cmd/cluster-apply.go

```diff
@@ -89,10 +93,10 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 		fmt.Printf("\nEnsuring that cluster controlplane is up to date.\n")
 
 		cu := controlplaneUpdater{
-			kubeconfigPath: kubeconfigPath,
-			assetDir:       assetDir,
-			ctxLogger:      *ctxLogger,
-			ex:             *ex,
+			kubeconfig: kubeconfig,
```

Comment:
You can just do

```go
cu := controlplaneUpdater{
		kubeconfig,
		assetDir,
		ctxLogger:      *ctxLogger,
		ex:             *ex,
}
```

Reply:
Right, but I think we should prefer used convention here, to avoid implicit coupling on struct fields order. Also, such change would be unrelated to this already large PR.

Reply:
Ok, sounds reasonable.